<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window1</class>
 <widget class="QWidget" name="window1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(252,157,154);</string>
     </property>
     <property name="title">
      <string>节点1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>温度数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node1_LB1">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>℃</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>湿度数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node1_LB2">
              <property name="text">
               <string>50</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>RH</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>光照数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node1_LB3">
              <property name="text">
               <string>200</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>LUX</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>电池电量：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node1_LB4">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>在线状态：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node1_LB5">
              <property name="text">
               <string>离线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(200,200,169);</string>
     </property>
     <property name="title">
      <string>节点2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>温度数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node2_LB1">
              <property name="text">
               <string>20</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>℃</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>湿度数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node2_LB2">
              <property name="text">
               <string>60</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>RH</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>光照数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node2_LB3">
              <property name="text">
               <string>300</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>LUX</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>电池电量：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node2_LB4">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>在线状态：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node2_LB5">
              <property name="text">
               <string>离线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(137,207,240);</string>
     </property>
     <property name="title">
      <string>节点3</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>温度数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node3_LB1">
              <property name="text">
               <string>30</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>℃</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>湿度数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node3_LB2">
              <property name="text">
               <string>70</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>RH%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_37">
              <property name="text">
               <string>光照数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node3_LB3">
              <property name="text">
               <string>400</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_39">
              <property name="text">
               <string>LUX</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_40">
              <property name="text">
               <string>电池电量：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node3_LB4">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_42">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_43">
              <property name="text">
               <string>在线状态：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="node3_LB5">
              <property name="text">
               <string>离线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_45">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>124</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>124</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/back.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
